gettsim/functions.py

```diff
@@ -0,0 +1,27 @@
+from gettsim.benefits.arbeitsl_geld import *  # noqa: F401, F403
```

Comment:
We only need this for the docs, right? If so, add docstring like:

```suggestion
"""Import all functions to a central place in order to automatically create the overview
in the documentation.

"""


from gettsim.benefits.arbeitsl_geld import *  # noqa: F401, F403
```

Reply:
Yes.

Reply:
Thinking about it: Does this not just duplicate the function loader? Would it be useful to add a test that checks for discrepancies?